telemetry/redis_ts.dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM redislabs/redistimeseries:1.4.5
```

Comment:
This can just be `Dockerfile` instead of `redis_ts.dockerfile`. Doesn't matter a ton, but since we only have one Dockerfile that's the convention